content/post/proficiency_v_creativity.md

```diff
@@ -0,0 +1,63 @@
+---
+author: Caitlin Moorman
+title: "Proficiency v. Creativity"
+cover: ""
+draft: true
+publishDate: "2020-06-01"
+tags:
+ - org
+ - teams
+ - management
+ - creativity
+---
+
+ I recently read [The Culture Code](https://www.indiebound.org/book/9780804176989) by Daniel Coyle, and among the many gems about building great organizational cultures, one idea stood out to me specifically for my own team: managing for proficiency versus managing for creativity. Proficiency-focused organizations must consistently and reliably perform clearly defined tasks, while creativity-focused organizations require consistent innovation and fresh ideas.
+
+Coyle argues that these two types of organizations require substantially different cultures to optimize for either proficiency or creativity. But successful data teams require both - so how do we lead teams that balance the two?
+
+<!--more-->
```

Comment:
the `<!--more-->` tag removes line breaks so you may want to add a trailing space to the first paragraph so there `. ` instead of just `.` when the paragraphs are combined